fzf-git.sh

```diff
@@ -50,6 +50,20 @@ __fzf_git_cat() {
   fi
 }
 
+__fzf_pager() {
+  local core_pager
+  core_pager=$(git config --get core.pager)
+  if [[ -n $FZF_PAGER ]]; then
+    echo "$FZF_PAGER"
```

Comment:
A configuration variable should have `FZF_GIT_` prefix by convention.

```suggestion
  if [[ -n $FZF_GIT_PAGER ]]; then
    echo "$FZF_GIT_PAGER"
```

See https://github.com/junegunn/fzf-git.sh/commit/bd8ac4ba4c9d7d12b34f7fa2b0d334f50cdb5254

Reply:
That makes sense; the same was applied to the function name as well.